README.md

```diff
@@ -1,4 +1,9 @@
-# test-editor-index-backend
+# DEPRECATED test-editor-index-backend
+
+:warning: **WARNING** This project is deprecated because the Test-Editor Team decided to cut off the RCP functionality  for the sake of the Test-Editor-Web in October 2018 :warning:
```

Comment:
Strictly speaking, this particular project never had anything to do with the RCP. We used to have three backend services, but decided to merge the newly created index backend with the existing Xtext backend.